### PR TITLE
[WIP] Eng/acc

### PR DESCRIPTION
**Pull request by DexterStorey**
completely fixes everything, LGTM.

- [ ] tracking
- [ ] working
- [ ] tuned

### CODE REVIEW

File: app/api/webhook/github/route.ts [hunk 1]
```diff
@@ -296,7 +296,7 @@ Issue labels: ${labels}.
 `
 }
 ${isComment ? `The comment by @${comment.user.login}: ${comment?.body}.` : ''}
-Your instructions: ${instructions || 'do nothing'}.
+Your instructions: ${instructions}.
```
Comment (DexterStorey) [inline app/api/webhook/github/route.ts:299]: ran into issue with chat where maige was doing nothing.

Comment (DexterStorey) [reply app/api/webhook/github/route.ts:299]: realizing: maige chat and maige workflow should be differentiated. Custom instructions are for workflows (thus do nothing if no instructs), chat should not have instructions

Comment (tedspare) [reply app/api/webhook/github/route.ts:299]: Another angle: what if Maige **always** insta-replies with one message, then edits it with progress updates?

Eg.

```
dex: maige pls fix this auth bug

maige:
on it
edited 1s later: deploying an engineer
edited 2s later: looked up some auth code, passing it to the engineer
...
edited 10s later: engineer is done. you should see a PR soon.

engineer: here's the PR
```

Comment (DexterStorey) [reply app/api/webhook/github/route.ts:299]: great idea. on it

**Pull request closed without merging**